hazelcast-client/src/main/java/com/hazelcast/client/impl/HazelcastClientInstanceImpl.java

```diff
@@ -555,6 +561,10 @@ public NearCacheManager getNearCacheManager() {
         return nearCacheManager;
     }
 
+    public ClientLoggingService getClientLoggingService() {
```

Comment:
I would return the interface 'LoggingService' and rename this method to getLoggingService.